.github/workflows/daily.yml

```diff
@@ -18,7 +18,13 @@ jobs:
         uses: actions/checkout@v3
       - name: Build and test all modified packages
         id: test
-        run: scripts/test/test_install.ps1
+        run: scripts/test/test_install.ps1 -all
+      - name: Upload log to artifacts
+        uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: failed_packages.txt
```

Comment:
I don't think this file will ever exist since we're not testing `flarevm.installer.vm`

Reply:
We may want to manually create this file like:

```
$failedPackages = "failed_packages.txt"

..[SNIP]...

if ($validExitCodes -notcontains $LASTEXITCODE) {
    Add-Content $failedPackages $package

...[SNIP]...
```

Reply:
it exists, we were already uploading it. Check the daily uploads: https://github.com/mandiant/VM-Packages/actions/runs/3467207358